src/provider/view/components/CircleButton/CircleButton.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../styles/_variables.scss';
+@import '../../styles/animations.scss';
+
+[class^="icon-"],
+[class*="icon-"] {
+    display: inline-block;
+    width: 100%;
+}
+
+$closeIcon: url('../../../../../res/provider/ui/image/shapes/Close.svg');
+$arrowDownIcon: url('../../../../../res/provider/ui/image/shapes/ArrowDown.svg');
+$check: url('../../../../../res/provider/ui/image/shapes/Check.svg');
+$minus: url('../../../../../res/provider/ui/image/shapes/Minus.svg');
+
+.icon {
+    $size: 18px;
+    background-color: var(--strip-background-color);
+    width: $size;
+    height: $size;
+    min-width: $size;
+    min-height: $size;
+    align-self: center;
+    display: inline-block;
+    align-content: center;
+    justify-content: center;
+    border-radius: $size;
+    cursor: pointer;
+    text-align: center;
+    transition: background-color 0.1s ease-in-out;
+
+    .image {
+        width: 100%;
+        height: 100%;
+        display: inline-block;
+        background-color: var(--accent-color);
+        background-blend-mode: multiply;
+        margin-left: -1px;
+        -webkit-mask: $closeIcon;
+        -webkit-mask-position: center center;
+        -webkit-mask-size: 70%;
+        -webkit-mask-repeat: no-repeat;
+    }
+
+    &:hover {
+        transition: background-color 0.2s ease-in-out;
+        background-color: var(--secondary-text-color-highlight);
+    }
+
+    &.large {
+        $size: 32px;
+        width: $size;
+        height: $size;
+    }
+
+    &.accept {
+        $color: #95D154;
+        background-color: $color;
+
+        .image {
+            -webkit-mask: $check;
+            -webkit-mask-position: center center;
+            -webkit-mask-size: 70%;
+            -webkit-mask-repeat: no-repeat;
+
+            background-color: var(--strip-background-color);
+        }
+
+        &:hover {
+            background-color: lighten(saturate($color, 40%), 10%);
+        }
+    }
+
+    &.cancel {
+        $color: #DF5353;
+        background-color: $color;
+
+        .image {
+            background-color: var(--strip-background-color);
+        }
+
+        &:hover {
+            background-color: lighten(saturate($color, 40%), 10%);
+        }
+    }
+
+    &.close {
+        &:hover {
+            background-color: rgb(233, 128, 128);
+        }
+    }
+
+    &.dismiss {
+        .image {
+            -webkit-mask: $minus;
```

Comment:
Is there an icon that better reflects "minimise"? I don't think the difference between `-` and `x` icons is particularly clear..?

Reply:
I went with a minus icon as that is the windows minimize icon. I could add a yellow color to the button on mouse over (like how the `x` has a red color on hover).

Reply:
Huh... That's something I never noticed.

The minimise icon in previous windows versions was always more of an "underscore", now it's more of a "hyphen". Let's not start doing OS-specific icons, though :-)

Reply:
It's also a `-` on MacOS & most Linux distros, guess it is universally agreed on now. 
`_` (underscore) wouldn't work with a circular button either.